src/test/java/com/exasol/glue/ittests/DataWritingIT.java

```diff
@@ -0,0 +1,151 @@
+package com.exasol.glue.ittests;
+
+import static com.exasol.matcher.ResultSetStructureMatcher.table;
+import static com.exasol.matcher.TypeMatchMode.NO_JAVA_TYPE_CHECK;
+import static org.apache.spark.sql.functions.col;
+import static org.apache.spark.sql.functions.date_format;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.startsWith;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.List;
+
+import com.exasol.dbbuilder.dialects.Table;
+import com.exasol.glue.ExasolValidationException;
+
+import org.apache.spark.sql.*;
+import org.hamcrest.Matcher;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Tag("integration")
+@Testcontainers
+class DataWritingIT extends BaseIntegrationTestSetup {
+    private static final int INTEGER_MIN = -2147483648;
```

Comment:
Use the constants from the standard library instead: https://docs.oracle.com/javase/7/docs/api/java/lang/Integer.html#MAX_VALUE